.woodpecker/.latest.yml

```diff
@@ -1,10 +1,13 @@
-pipeline:
-  build:
-    image: plugins/docker
+steps:
+  build-and-push:
+    image: woodpeckerci/plugin-docker-buildx
     settings:
-      repo: ${CI_REPO}
+      repo: "${CI_REPO_OWNER%%io}/${CI_REPO_NAME}"
```

Comment:
I don't think this part needs to be changed. That seems specific for repos in the redpencilio org.